src/_examples/index.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: page
+title: Examples
+---
+
+# Module-specific Examples
+
+<ul>
+{% for example in site.examples %}
+  {% if example.module %}
+    <li><a href="{{ example.url }}">{{ example.module }}</a></li>
+  {% endif %}
+{% endfor %}
+</ul>
+
+# Standalone Projects
+
+These projects are using Workbox for their service worker:
+
+- iFixit PWA (`gulp`-based build process)
```

Comment:
FYI, https://ifixit-pwa.appspot.com/ and https://github.com/GoogleChrome/application-shell/tree/ifixit-pwa/ifixit-pwa have been updated to use the current `0.0.1` published libraries.